Refresh.GameServer/Configuration/GameServerConfig.cs

```diff
@@ -9,16 +9,16 @@ namespace Refresh.GameServer.Configuration;
 [SuppressMessage("ReSharper", "RedundantDefaultMemberInitializer")]
 public class GameServerConfig : Config
 {
-    public override int CurrentConfigVersion => 13;
+    public override int CurrentConfigVersion => 14;
     public override int Version { get; set; } = 0;
 
     protected override void Migrate(int oldVer, dynamic oldConfig) {}
 
     public string LicenseText { get; set; } = "Welcome to Refresh!";
 
-    public AssetSafetyLevel MaximumAssetSafetyLevel { get; set; } = AssetSafetyLevel.SafeMedia;
+    public AssetFlags BlockedAssetFlags { get; set; } = AssetFlags.Dangerous | AssetFlags.Modded;
     /// <seealso cref="GameUserRole.Trusted"/>
-    public AssetSafetyLevel MaximumAssetSafetyLevelForTrustedUsers { get; set; } = AssetSafetyLevel.SafeMedia;
+    public AssetFlags BlockedAssetFlagsForTrustedUsers { get; set; } = AssetFlags.Dangerous | AssetFlags.Modded;
```

Comment:
These need to have some sort of migration step.

Additionally we should provide documentation on setting this value, since I don't imagine bitwise values are easy to work with from a user's perspective. Maybe a class with boolean flags that get converted to the bitwise value instead?